tests/manage/z_cluster/nodes/test_node_kernel_crash_ceph_fsync.py

```diff
@@ -0,0 +1,114 @@
+import logging
+import random
+import pytest
+
+from time import sleep
+from ocs_ci.ocs import constants, node
+from ocs_ci.framework.testlib import E2ETest, tier1, bugzilla, polarion_id
+from ocs_ci.ocs.exceptions import ResourceWrongStatusException
+from ocs_ci.ocs.node import get_worker_nodes
+from concurrent.futures import ThreadPoolExecutor
+from ocs_ci.helpers import helpers
+
+log = logging.getLogger(__name__)
+
+
+@tier1
```

Comment:
The test is not introducing any kernel crash explicitly. It runs fsync operations while create delete files are going on.  Ideally these operations are basic operations and can be in tier1. Since tier1 gets run very frequently than other tiers, we have more opportunities to catch serious bugs like kernel crash if we added it as part of tier1.